app/intervention/[id]/(tabs)/means/index.tsx

```diff
@@ -50,19 +52,56 @@ export default function Means() {
     (payload) => {
       switch (payload.eventType) {
         case 'INSERT':
-          onInsert(payload.new as any);
+          onInsertMeans(payload.new as any);
           break;
 
         case 'UPDATE':
-          onUpdate(payload.new as any);
+          onUpdateMeans(payload.new as any);
```

Comment:
```suggestion
          onMeanUpdated(payload.new as any);
```

Même question qu'au dessus